internal/monitors/collectd/kong/kong.tmpl

```diff
@@ -0,0 +1,74 @@
+LoadPlugin "python"
+<Plugin python>
+  ModulePath "{{ pluginRoot }}/kong"
+  LogTraces true
+  Import "kong_plugin"
+  <Module kong_plugin>
+    ExtraDimension "monitorID" "{{.MonitorID}}"
+    {{with .URL}}
+    URL "{{renderValue . $ }}"
+    {{end}}{{with .Verbose}}
+    Verbose {{.}}
+    {{end}}{{with .Name}}
+    Name "{{.}}"
+    {{end}}{{with $x := .AuthHeader}}
+    AuthHeader "{{$x.HeaderName}}" "{{$x.Value}}"
+    {{end}}{{with .VerifyCerts}}
+    VerifyCerts {{.}}
+    {{end}}{{with .CABundle}}
+    CABundle "{{.}}"
+    {{end}}{{with .ClientCert}}
+    ClientCert "{{.}}"
+    {{end}}{{with .ClientCertKey}}
+    ClientCertKey "{{.}}"
+    {{end}}{{with .IntervalSeconds}}
+    Interval {{.}}
+    {{end}}{{range $l := .Metrics -}}
+    Metric "{{$l.MetricName}}" {{$l.ReportBool}}
+    {{end}}{{with .ReportAPIIDs}}
+    ReportAPIIDs {{.}}
```

Comment:
Do all of these `Report*` options default to false in the plugin?  Because if not, I think they are only going to be configured if the user provided `true`.

Reply:
No, they have various defaults (https://github.com/signalfx/collectd-kong/blob/master/README.md) and I've confirmed desired fields propagate for configuration changes and plugin defaults are used.

defaults are those of collectd, since I'm not specifying agent defaults:
```yaml
    reportRouteIds: false  # true by default
    reportHttpMethods: false  # true by default
    reportStatusCodes: true   # true by default
    reportStatusCodeGroups: false  # true by default
```

```
[0004]  INFO report_api_ids: True collectdInstance=global (default)
[0004]  INFO report_api_names: True collectdInstance=global (default)
[0004]  INFO report_http_methods: False collectdInstance=global (agent.yml)
[0004]  INFO report_route_ids: False collectdInstance=global (agent.yml)
[0004]  INFO report_service_ids: True collectdInstance=global (default)
[0004]  INFO report_service_names: True collectdInstance=global (default)
[0004]  INFO report_status_code_groups: False collectdInstance=global (agent.yml)
[0004]  INFO report_status_codes: True collectdInstance=global (agent.yml)
```